dv-export/python/query-changes/command_handler.py

```diff
@@ -9,6 +11,16 @@
 logger = logging.getLogger(__name__)
 
 
+class QueryMode(Enum):
```

Comment:
Query mode is the name of a pre-existing list query parameter that refers to how the query should behave when there are invalid columns. 
It's tempting to consider just exposing record mode as the sample parameter also, but the values you have are more intuitive than those for the option so perhaps something like 'extraction mode'?

Reply:
Renamed to ExtractionMode.